src/main/java/me/partlysanestudios/partlysaneskies/CustomMainMenu.java

```diff
@@ -461,6 +461,12 @@ public static void setMainMenuInfo(Request request) {
             return;
         }
 
+        try {
+            Prank.Companion.setPrankKillSwitch(object.get("prank_sound").getAsBoolean());
+        } catch(NullPointerException | IllegalStateException e){
+            Utils.sendClientMessage("Something went wrong while trying to get a safety feature. Please report this in our discord server.");
```

Comment:
I'm gonna be kinda annoying and ask that this is removed. We're going to get just a bunch of people from India where GitHub is blocked

Reply:
You can just e.printStackTrace()